core/templates/dev/head/components/rating_display.html

```diff
@@ -1,5 +1,5 @@
 <script type="text/ng-template" id="components/ratingSummary">
   <span style="<[getCursorStyle()]>" ng-mouseleave="leaveArea()">
-    <span ng-if="ratingValue || isEditable" ng-repeat="star in stars" class="fa <[star.cssClass]> protractor-test-rating-star" ng-click="clickStar(star.value)" ng-mouseenter="enterStar(star.value)"></span>
+    <span ng-if="ratingValue || isEditable" ng-repeat="star in stars" class="fa <[star.cssClass]> protractor-test-rating-star oppia-rating-star" ng-click="clickStar(star.value)" ng-mouseenter="enterStar(star.value)"></span>
```

Comment:
Optional nit: It is a small thing, but any chance you could have the protractor-test-* class always come last? In a way it's the least important one and I think it's nice to have a consistent pattern throughout of it being the last in the list.

Ditto elsewhere.